### PR TITLE
a little change in ddpg.py

### PR DESCRIPTION
**Pull request by kxxwz**
volatile was removed and now has no effect. Use `with torch.no_grad():` instead.

### CODE REVIEW

**Comment by songanz** [comment]
I want to know, in the original version, the author set the `next_q_values.volatile=False`, and then do `value_loss = criterion(q_batch, target_q_batch)`
`value_loss.backward()`
So, in the original version, is he trying to do the backward() on both the target critic network and the critic network?

**Comment by CoolPhilChen** [comment]
I don't think the target networks will be updated by policy gradient.

**Comment by kxxwz** [comment]
> I want to know, in the original version, the author set the `next_q_values.volatile=False`, and then do `value_loss = criterion(q_batch, target_q_batch)`
> `value_loss.backward()`
> So, in the original version, is he trying to do the backward() on both the target critic network and the critic network?

I don't think the target networks will be updated by policy gradient.
